lib/spack/spack/cmd/verify.py

```diff
@@ -67,16 +67,65 @@ def setup_parser(subparser: argparse.ArgumentParser):
 
     arguments.add_common_arguments(libraries_subparser, ["constraint"])
 
+    versions_subparser = sp.add_parser(
+        "versions", help=verify_versions.__doc__, description=verify_versions.__doc__
+    )
+    versions_subparser.add_argument("-a", "--all", help="verify versions for all packages")
```

Comment:
```suggestion
    versions_subparser.add_argument("-a", "--all", help="check installed packages for deprecated/unverified versions")
```

Personal preference, but I think we should be more specific here in the help message. What does it mean to verify a version is going to be a common question. Does it mean verify it is still installed and not broken?